builder/infer_bounds.cc

```diff
@@ -508,7 +520,14 @@ class slide_and_fold_storage : public node_mutator {
     }
     var orig_min(ctx, ctx.name(op->sym) + ".min_orig");
 
-    loops.push_back({op->sym, orig_min, bounds(orig_min, op->bounds.max), op->step});
+    auto last_buffer_bounds = current_buffer_bounds();
+    loops.emplace_back(op->sym, orig_min, bounds(orig_min, op->bounds.max), op->step);
+    for (int ix = 0; ix < (int)last_buffer_bounds.size(); ix++) {
```

Comment:
I think this is just `current_buffer_bounds() = last_buffer_bounds`. If not, we should add `symbol_map::operator=` so it works.

Reply:
Done.